livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -64,6 +64,7 @@ public void open() {
 
   @Override
   public void close() {
+    livyHelper.closeSession(userSessionMap, "spark");
```

Comment:
shouldn't here the session kind be something other than "spark", since that is used in the LivySparkInterpreter?